src/os/ui/featureedit.js

```diff
@@ -1227,28 +1298,59 @@ os.ui.FeatureEditCtrl.prototype.saveGeometry_ = function(feature) {
         os.feature.createEllipse(feature, true);
       } else {
         // clear ellipse fields on the feature
-        feature.set(os.Fields.SEMI_MAJOR, undefined);
-        feature.set(os.Fields.SEMI_MINOR, undefined);
-        feature.set(os.Fields.SEMI_MAJOR_UNITS, undefined);
-        feature.set(os.Fields.SEMI_MINOR_UNITS, undefined);
-        feature.set(os.Fields.ORIENTATION, undefined);
-        feature.set(os.data.RecordField.ELLIPSE, undefined);
-        feature.set(os.data.RecordField.LINE_OF_BEARING, undefined);
+        feature.set(os.Fields.SEMI_MAJOR, undefined, true);
+        feature.set(os.Fields.SEMI_MINOR, undefined, true);
+        feature.set(os.Fields.SEMI_MAJOR_UNITS, undefined, true);
+        feature.set(os.Fields.SEMI_MINOR_UNITS, undefined, true);
+        feature.set(os.Fields.ORIENTATION, undefined, true);
+        feature.set(os.data.RecordField.ELLIPSE, undefined, true);
+        feature.set(os.data.RecordField.LINE_OF_BEARING, undefined, true);
       }
 
       if (!this.isFeatureDynamic() && (this.showIcon() || this.showCenterIcon()) && this['iconRotation'] != null) {
-        feature.set(os.style.StyleField.SHOW_ROTATION, true);
-        feature.set(os.Fields.BEARING, this['iconRotation'] % 360);
-        feature.set(os.style.StyleField.ROTATION_COLUMN, os.Fields.BEARING);
+        feature.set(os.style.StyleField.SHOW_ROTATION, true, true);
+        feature.set(os.Fields.BEARING, this['iconRotation'] % 360, true);
+        feature.set(os.style.StyleField.ROTATION_COLUMN, os.Fields.BEARING, true);
       } else {
-        feature.set(os.Fields.BEARING, undefined);
-        feature.set(os.style.StyleField.SHOW_ROTATION, false);
-        feature.set(os.style.StyleField.ROTATION_COLUMN, '');
+        feature.set(os.Fields.BEARING, undefined, true);
+        feature.set(os.style.StyleField.SHOW_ROTATION, false, true);
+        feature.set(os.style.StyleField.ROTATION_COLUMN, '', true);
       }
     }
-  } else if (this.originalGeometry) {
+  } else if (this.originalGeometry && feature.getGeometry() === this.originalGeometry) {
```

Comment:
This change is preventing non-point geometries from displaying on the map when creating a new place. The preview feature geometry is never set, so the second condition is always false.